script/publish_public_data_json

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+"""
+Export query result data as JSON to a publicly accessible bucket.
+
+Data of the query is exported if "json_export" is set in
+the corresponding metadata file.
+"""
+
+from argparse import ArgumentParser
+import os
+import sys
+import re
+
+from google.cloud import bigquery
+
+# sys.path needs to be modified to enable package imports from parent
+# and sibling directories. Also see:
+# https://stackoverflow.com/questions/6323860/sibling-package-imports/23542795#23542795
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Maybe it's time to reconsider the directory structure.

Reply:
Long-term, I think this repo is going to need a significant refactor where we pull all the python logic inside the `bigquery_etl` module and the scripts become light entrypoints into that codebase. But that's definitely outside the scope of this PR.